buildpipeline/perf-pipeline.groovy

```diff
@@ -354,15 +317,15 @@ def innerLoopTests = [:]
 ['x64', 'x86'].each { arch ->
     ['full_opt'].each { opt_level ->
         [false].each { isBaseline ->
-            validTestFolders.each { benchmark ->
+            [0,1,2,3,4,5].each { slice ->
```

Comment:
Silly question - is there a range function to build this?  It's more stylistic than anything else.

Reply:
There is, didn't think to look for it.

Reply:
Ok, so there is an issue with using a range, and that is that it is non serializable and with pipelines, everything has to be serializable. so I am going to have to keep it as the list.